src/markdown-pages/nr1-cli/nr1-nerdpack.mdx

```diff
@@ -0,0 +1,142 @@
+---
+path: '/explore-docs/nr1-nerdpack'
+title: 'New Relic One CLI Nerdpack commands'
+template: 'GuideTemplate'
+description: 'An overview of the CLI commands you can use to set up your New Relic One Nerdpacks.'
+tileShorthand:
+  title: 'Nerdpack CLI commands'
+  description: 'Set up your Nerdpacks with the New Relic One CLI.'
+redirects:
+---
+<Intro>
+
+To set up your Nerdpacks, use the commands below. You can click any command to see its usage and options, among other details.
+
+</Intro>
+
+<br/>
+
+| Command | Description |
+|---|---|
+| [`nr1 nerdpack:clone`](#clone) | Clones a Nerdpack from a git repository. |
+| [`nr1 nerdpack:serve`](#serve) | Serves your Nerdpack for testing and development purposes. |
+| [`nr1 nerdpack:uuid`](#uuid) | Shows or regenerates the UUID of a Nerdpack.  |
+| [`nr1 nerdpack:publish`](#publish) | Publishes your Nerdpack to New Relic. |
+| [`nr1 nerdpack:deploy`](#deploy) | Deploys a specific Nerdpack version to a specific channel. |
+
+# Command details
+
+***
+
+## `nr1 nerdpack:clone` 
+### Clone an existing Nerdpack
+Duplicates an existing Nerdpack onto your local computer. You can clone an open source Nerdpack from [our GitHub repository](https://opensource.newrelic.com/explore-projects/). After choosing a git repository, this command performs the following actions so that you can start using the Nerdpack:
+1. Clones the repository.
+2. Sets the repository as remote upstream.
+3. Installs all of its dependencies (using npm).
+4. Generates a new UUID using your profile, and commits it.
+
+### Usage
+`$ nr1 nerdpack:clone OPTION` 
+
+### Options
+| | |
+|---|---|
+|  `-r, --repo=REPO`  | Repository location (either an HTTPS or SSH path). **(Required)** |
+|  `-p, --path=PATH`  | Determines the directory to clone to (defaults to the repository name). |
+|  `-f, --force`      |   Replaces destination folder if it exists. |
+|  `--profile=PROFILE`| The authentication profile you want to use. |
+|  `--verbose`        | Adds extra information to the output. |
+
+<br/>
+
+*** 
+
+## `nr1 nerdpack:serve`
+### Serve your Nerdpack locally
+Launches a server with your Nerdpack locally on your [New Relic One platform](https://one.newrelic.com/?nerdpacks=local), where it can be tested live. 
```

Comment:
```suggestion
Launches a server with your Nerdpack locally on the [New Relic One platform](https://one.newrelic.com/?nerdpacks=local), where it can be tested live. 
```